wemake_python_styleguide/visitors/ast/blocks.py

```diff
@@ -151,6 +158,13 @@ def visit_locals(self, node: Union[AnyAssign, ast.arg]) -> None:
 
     # Utils:
 
+    def _is_function_overload(self, node: ast.AST):
```

Comment:
This can actually be a function, not a method. `self._overload_exceptions` can be used as a regular set.